valorant-tracker/src/pages/home/Home.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function Home() {
+  return (
+    <form>
+      <label>Username
+        <input type="text" />
+      </label>
+      <label>Password
```

Comment:
We are taking #tagline, not the password. Also, password input fields must be of type 'password' by default to hide the characters.